groups/script.js

```diff
@@ -22,6 +23,19 @@ const userIsNotVerifiedText = document.querySelector('.not-verified-tag');
 const params = new URLSearchParams(window.location.search);
 const searchValue = getSearchValueFromURL();
 const isDev = params.get(DEV_FEATURE_FLAG) === 'true';
+const dropdownContainer = document.getElementById('dropdown_container');
+
+//Dropdown
+const dropdown_main = document.getElementById('dropdown_main');
+const dropdown_txt = document.getElementById('sortby_text');
+function toggleDropDown() {
+  dropdown_main.classList.toggle('show_filter');
+}
+dropdown_txt.addEventListener('click', toggleDropDown);
+if (isDev) {
+  dropdownContainer.classList.remove('hidden');
+  dropdown_main.addEventListener('click', onDropdownClick);
```

Comment:
Why not using camelCase for some variables?

Reply:
Done